src/CMakeLists.txt

```diff
@@ -428,9 +431,11 @@ target_link_libraries(${EXECUTABLE_CPPREGIONTEST} ${COMMON_LIBS})
 set_target_properties(${EXECUTABLE_CPPREGIONTEST} PROPERTIES COMPILE_FLAGS ${COMMON_COMPILE_FLAGS})
 set_target_properties(${EXECUTABLE_CPPREGIONTEST}
                       PROPERTIES LINK_FLAGS "${COMMON_LINK_FLAGS}")
+if(UNIX) # or UNIX like (i.e. APPLE and CYGWIN)
```

Comment:
Why you removed `cpp_region_test` from Windows? It is working fine in my Windows machine.